Tasks/RunLoadTest/task.json

```diff
@@ -0,0 +1,89 @@
+﻿{
+  "id": "9E9DB38A-B40B-4C13-B7F0-31031C894C22",
+  "name": "CloudLoadTest",
+  "friendlyName": "Cloud-based Load Test with VSO",
+  "description": "Runs the load test in cloud; Powered by VS Online.",
+  "category": "Test",
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 1,
+    "Minor": 0,
+    "Patch": 1
+  },
+  "demands": [
+    "msbuild"
+  ],
+  "groups": [
+    {
+      "name": "advanced",
+      "displayName": "Advanced",
+      "isExpanded": false
+    }
+  ],
+  "inputs": [
+    {
+      "name": "TestDrop",
+      "type": "filePath",
+      "label": "Load test files folder",
+      "defaultValue": "$(Build.SourcesDirectory)\\loadtest",
+      "required": true,
+      "helpMarkDown": "Relative path from build source directory on the agent under which the load test solution output is placed."
+    },
+    {
+      "name": "LoadTest",
+      "type": "string",
+      "label": "Load test file",
+      "defaultValue": "loadtest1.loadtest",
+      "required": true,
+      "helpMarkDown": "Relative path to the load test filename under the test drop folder defined above."
+    },
+    {
+      "name": "TestSettings",
+      "type": "string",
+      "label": "Test settings file",
+      "defaultValue": "local.testsettings",
+      "required": true,
+      "helpMarkDown": "Relative path to the test settings filename under the test drop folder defined above."
+    },
+    {
+      "name": "ThresholdLimit",
+      "type": "string",
+      "label": "Number of permissible threshold violations",      
+      "required": false,
+      "helpMarkDown": "Number of threshold violations above which the load test outcome is considered unsucessful."
+    },
+    {
+      "name": "VsoAccountUrl",
+      "type": "string",
```

Comment:
As CLT is VSO only service, we need the URL and creds when running on-premises. There is no easy way to determine the CLT service endpoint, as location service REST APIs are not available. On a non-hybrid scenario, ideally we want to make use of PAT, which is pending work.